### PR TITLE
Micro with justice

### PR DESCRIPTION
**Pull request by Colin-Tel**
## About the PR

🧑‍⚖️  

## Media

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

idk lol

### CODE REVIEW

**Comment by deltanedas** [comment]
real...

**Comment by deltanedas** [comment]
MISINPUT

File: Resources/Locale/en-US/deltav/misc/pda.ftl [hunk 1]
```diff
@@ -0,0 +1,4 @@
+ent-HoSPDA = head of security pda
```
Comment (deltanedas) [inline Resources/Locale/en-US/deltav/misc/pda.ftl]: why is this here

File: Resources/Prototypes/DeltaV/Catalog/Fills/Lockers/chiefjustice.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    - id: ClothingUniformJumpsuitChiefJusticeWhite
+    - id: PaperStationWarrant
+      amount: 10
+    - id: BoxPDAJustice
+    - id: BoxEncryptionKeyJustice
+    - id: ChiefJusticeIDCard
+    - id: DoorRemoteJustice
+    - id: Gavel
+    - id: RubberStampChiefJustice
+    - id: LunchboxCommandFilledRandom # Delta-V Lunchboxes!
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Catalog/Fills/Lockers/chiefjustice.yml]: its not an upstream file :trollface:

File: Resources/Prototypes/DeltaV/Entities/Clothing/Ears/headsets.yml [hunk 3]
```diff
@@ -42,6 +42,45 @@
...
+  description: The headset used by the chief justice.
+  components:
+  - type: ContainerFill
+    containers:
+      key_slots:
+      - EncryptionKeyJustice
+      - EncryptionKeyPrison
+      - EncryptionKeySecurity
+      - EncryptionKeyCommon
+      - EncryptionKeyCommand
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Entities/Clothing/Ears/headsets.yml]: max is 4 so wont this overfill it

**Review by deltanedas — DISMISSED** [review]

File: Resources/Locale/en-US/deltav/misc/pda.ftl (hunk 1 above)
Comment (Colin-Tel) [reply Resources/Locale/en-US/deltav/misc/pda.ftl]: idk, it's a change from the Justice Dept PR

File: Resources/Prototypes/DeltaV/Catalog/Fills/Lockers/chiefjustice.yml (hunk 2 above)
Comment (Colin-Tel) [reply Resources/Prototypes/DeltaV/Catalog/Fills/Lockers/chiefjustice.yml]: haha, another Justice PR change

File: Resources/Prototypes/DeltaV/Entities/Clothing/Ears/headsets.yml (hunk 3 above)
Comment (Colin-Tel) [reply Resources/Prototypes/DeltaV/Entities/Clothing/Ears/headsets.yml]: I dunno lol

**Review by Axionyxx — DISMISSED** [review]

**Review by IamVelcroboy — APPROVED** [review]
